### ISSUE TITLE
post processor error

### ISSUE THREAD

**Issue by mikecali** [mitchellh/packer#3703]
Hi,

I am getting an error trying to build an OS using vsphere post-processors. Trying other post-processors were not an issue. 

Below are the details:

FOR FEATURES:

post-processor = vsphere

FOR BUGS:

Describe the problem and include the following information:
- Packer Version : 0.10.1
- Host platform: Ubuntu 16.04
- Vsphere: 5.5
- https://gist.github.com/mikecali/a68ed25ba16f9d608f6a7be5a8cf1dd7

```
Build 'vmware-iso' errored: 1 error(s) occurred:
- Post-processor failed: Failed: exit status 1
  Stdout: Opening VMX source: packer-rhel-7.0-x86_64-acc-c-vmware/rhel-7.0-x86_64-acc-c.vmx
  Opening OVF target: "vi://Administrator%40vsphere.local:xxxxx%21@172.17.203.6/xGenITO-ALABS/host/G5-Cluster"
  Writing OVF package: "vi://Administrator%40vsphere.local:xxxxx%21@172.17.203.6/xGenITO-ALABS/host/G5-Cluster"/"packer-rhel-7.0-acc-x86_64-vmware-20160710215542"/"packer-rhel-7.0-acc-x86_64-vmware-20160710215542".ovf
  Transfer Failed  
  Error: System error: (std::exception)
  Deleting directory tree below: "vi:
  Completed with errors
```

**Comment by rickard-von-essen** [comment]
Please gist the full log.

**Comment by mikecali** [comment]
Here is the complete log.
Note that using ovftool I can transfer the image to Vsphere.

[log_packer.txt](https://github.com/mitchellh/packer/files/370332/log_packer.txt)

**Comment by mwhooker** [comment]
```
2016/07/19 22:45:14 ui: [0;32m    vmware-iso (vagrant): Compressing: rhel-7.0-x86_64-acc-c.vmxf[0m
2016/07/19 22:45:16 Flagging to keep original artifact from post-processor 'vagrant'
2016/07/19 22:45:16 ui: [1;32m==> vmware-iso: Running post-processor: vsphere[0m
2016/07/19 22:45:16 ui: [0;32m    vmware-iso (vsphere): Uploading packer-rhel-7.0-x86_64-acc-c-vmware/rhel-7.0-x86_64-acc-c.vmx to vSphere[0m
2016/07/19 22:45:16 ui: [0;32m    vmware-iso (vsphere): Uploading packer-rhel-7.0-x86_64-acc-c-vmware/rhel-7.0-x86_64-acc-c.vmx to vSphere[0m
2016/07/19 22:45:16 packer: 2016/07/19 22:45:16 Starting ovftool with parameters: --noSSLVerify=true --acceptAllEulas --name="packer-rhel-7.0-acc-x86_64-vmware-20160719102611" --datastore="datastore1" --diskMode="thin" --network="" --vmFolder="" packer-rhel-7.0-x86_64-acc-c-vmware/rhel-7.0-x86_64-acc-c.vmx "vi://Administrator%40vsphere.local:Vcenter12%21@172.17.203.6/NextGenITO-ALABS/host/G5-Cluster"
2016/07/19 22:45:22 ui error: [1;31mBuild 'vmware-iso' errored: 1 error(s) occurred:

* Post-processor failed: Failed: exit status 1
Stdout: Opening VMX source: packer-rhel-7.0-x86_64-acc-c-vmware/rhel-7.0-x86_64-acc-c.vmx
Opening OVF target: "vi://Administrator%40vsphere.local:Vcenter12%21@172.17.203.6/NextGenITO-ALABS/host/G5-Cluster"
Writing OVF package: "vi://Administrator%40vsphere.local:Vcenter12%21@172.17.203.6/NextGenITO-ALABS/host/G5-Cluster"/"packer-rhel-7.0-acc-x86_64-vmware-20160719102611"/"packer-rhel-7.0-acc-x86_64-vmware-20160719102611".ovf

Transfer Failed                       
Error: System error: (std::exception)
Deleting directory tree below: "vi:
Completed with errors
[0m
2016/07/19 22:45:22 Builds completed. Waiting on interrupt barrier...
2016/07/19 22:45:22 machine readable: error-count []string{"1"}
2016/07/19 22:45:22 ui error: 
==> Some builds didn't complete successfully and had errors:
2016/07/19 22:45:22 machine readable: vmware-iso,error []string{"1 error(s) occurred:\n\n* Post-processor failed: Failed: exit status 1\nStdout: Opening VMX source: packer-rhel-7.0-x86_64-acc-c-vmware/rhel-7.0-x86_64-acc-c.vmx\nOpening OVF target: \"vi://Administrator%40vsphere.local:Vcenter12%21@172.17.203.6/NextGenITO-ALABS/host/G5-Cluster\"\nWriting OVF package: \"vi://Administrator%40vsphere.local:Vcenter12%21@172.17.203.6/NextGenITO-ALABS/host/G5-Cluster\"/\"packer-rhel-7.0-acc-x86_64-vmware-20160719102611\"/\"packer-rhel-7.0-acc-x86_64-vmware-20160719102611\".ovf\n\rTransfer Failed                       \nError: System error: (std::exception)\nDeleting directory tree below: \"vi:\nCompleted with errors\n"}
2016/07/19 22:45:22 ui error: --> vmware-iso: 1 error(s) occurred:

* Post-processor failed: Failed: exit status 1
Stdout: Opening VMX source: packer-rhel-7.0-x86_64-acc-c-vmware/rhel-7.0-x86_64-acc-c.vmx
Opening OVF target: "vi://Administrator%40vsphere.local:Vcenter12%21@172.17.203.6/NextGenITO-ALABS/host/G5-Cluster"
Writing OVF package: "vi://Administrator%40vsphere.local:Vcenter12%21@172.17.203.6/NextGenITO-ALABS/host/G5-Cluster"/"packer-rhel-7.0-acc-x86_64-vmware-20160719102611"/"packer-rhel-7.0-acc-x86_64-vmware-20160719102611".ovf

Transfer Failed                       
Error: System error: (std::exception)
Deleting directory tree below: "vi:
Completed with errors
2016/07/19 22:45:22 ui: 
==> Builds finished but no artifacts were created.
2016/07/19 22:45:22 waiting for all plugin processes to complete...
2016/07/19 22:45:22 /home/mikecali/packer/packer: plugin process exited
2016/07/19 22:45:22 /home/mikecali/packer/packer: plugin process exited
2016/07/19 22:45:22 /home/mikecali/packer/packer: plugin process exited
2016/07/19 22:45:22 /home/mikecali/packer/packer: plugin process exited
2016/07/19 22:45:22 /home/mikecali/packer/packer: plugin process exited
2016/07/19 22:45:22 /home/mikecali/packer/packer: plugin process exited
2016/07/19 22:45:22 /home/mikecali/packer/packer: plugin process exited
```

**Comment by mwhooker** [comment]
this was fixed in #3321
